docs/source/upgrade.rst

```diff
@@ -13,18 +13,23 @@ Dropping CUDA 9.2 / 10.0 / 10.1 Support
 CUDA 10.1 or earlier is no longer supported.
 Use CUDA 10.2 or later.
 
-Dropping NCCL v2.4 Support
------------------------------
+Dropping NCCL v2.4 / v2.6 / v2.7 Support
+----------------------------------------
 
-NCCL v2.4 is no longer supported.
+NCCL v2.4, v2.6, and v2.7 are no longer supported.
 
 Dropping Python 3.6 Support
 ---------------------------
 
 Python 3.6 is no longer supported.
 
-Changes in :class:`cupy.cuda.Stream` Behavior
----------------------------------------------
+Dropping NumPy 1.17 Support
+---------------------------
+
+NumPy 1.17 is not longer supported.
+
+Changes in :class:`cupy.cuda.Stream` and :class:`cupy.cuda.Device` Behavior
```

Comment:
Could you split the Device and Stream section?
Device things are more important for users (compared to Stream things) so I don't want to mix them.

```suggestion
Changes in :class:`cupy.cuda.Device` and :class:`cupy.cuda.Stream` Behavior
```